components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/authcontext/DefaultClaimsRetrieverTest.java

```diff
@@ -0,0 +1,242 @@
+
+    DefaultClaimsRetriever defaultClaimsRetriever;
+
+    public static final String DEFAULT_DIALECT_URI = "http://wso2.org/claims";
+
+    @Mock
+    private OAuthServerConfiguration mockedOAuthServerConfiguration;
+
+    @Mock
+    private OAuthComponentServiceHolder mockedOAuthComponentServiceHolder;
+
+    @Mock
+    private RealmService mockedRealmService;
+
+    @Mock
+    private UserRealm mockedUserRealm;
+
+    @Mock
+    private UserStoreManager mockedUserStoreManager;
+
+    @Mock
+    private ClaimManager mockedClaimManager;
+
+    @Mock
+    private ClaimMapping mockedClaimMapping;
+
+    @BeforeTest
+    public void setUp() {
+        defaultClaimsRetriever = new DefaultClaimsRetriever();
+    }
+
+    private ClaimMapping[] getSampleClaimMapping() {
+        String coreClaimUri1;
+        String testMappedAttributesCore1;
+
+        String coreClaimUri2;
+        String testMappedAttributesCore2;
+
+        String userClaimUri1;
+        String testMappedAttributesUser1;
+
+        String userClaimUri2;
+        String testMappedAttributesUser2;
+
+        Claim claim1 = new Claim();
+        Claim claim2 = new Claim();
+        Claim claim3 = new Claim();
+        Claim claim4 = new Claim();
+
+        coreClaimUri1 = "testCoreClaimURI1";
+        claim1.setClaimUri(coreClaimUri1);
+        testMappedAttributesCore1 = "MappedAttributesCore1";
+
+        coreClaimUri2 = "testCoreClaimURI2";
+        claim2.setClaimUri(coreClaimUri2);
+        testMappedAttributesCore2 = "MappedAttributesCore2";
+
+        userClaimUri1 = "testUserClaimURI1";
+        claim3.setClaimUri(userClaimUri1);
+        testMappedAttributesUser1 = "MappedAttributesUser1";
+
+        userClaimUri2 = "testUserClaimURI2";
+        claim4.setClaimUri(userClaimUri2);
+        testMappedAttributesUser2 = "MappedAttributesUser2";
+
+        ClaimMapping cMap1 = new ClaimMapping(claim1, testMappedAttributesCore1);
+        ClaimMapping cMap2 = new ClaimMapping(claim2, testMappedAttributesCore2);
+        ClaimMapping cMap3 = new ClaimMapping(claim3, testMappedAttributesUser1);
+        ClaimMapping cMap4 = new ClaimMapping(claim4, testMappedAttributesUser2);
+
+        ClaimMapping[] coreClaims = new ClaimMapping[]{cMap1, cMap2, cMap3, cMap4};
+
+
+        return coreClaims;
+    }
+
+    @Test
+    public void testInit() throws Exception {
+        // Subject is null.
+        assertFalse(Boolean.parseBoolean(defaultClaimsRetriever.DEFAULT_DIALECT_URI));
+
+        // Subject is not null.
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedOAuthServerConfiguration);
+        when(mockedOAuthServerConfiguration.getConsumerDialectURI()).thenReturn("consumerDialectURI");
+        assertFalse(Boolean.parseBoolean(mockedOAuthServerConfiguration.getConsumerDialectURI()));
+    }
+
+    @Test
+    public void testGetClaims() throws Exception {
+        SortedMap<String, String> claimValues;
```

Comment:
We haven't used this variable anywhere within this method. If it is not used remove it.

Reply:
Removed the variable